Libraries/LibCrypto/BigFraction/BigFraction.cpp

```diff
@@ -198,9 +245,7 @@ String BigFraction::to_string(unsigned rounding_threshold) const
 
     auto const number_of_digits = [](auto integer) {
         unsigned size = 1;
-        for (auto division_result = integer.divided_by(UnsignedBigInteger { 10 });
-            division_result.remainder == UnsignedBigInteger { 0 } && division_result.quotient != UnsignedBigInteger { 0 };
-            division_result = division_result.quotient.divided_by(UnsignedBigInteger { 10 })) {
+        for (auto division_result = integer.divided_by(UnsignedBigInteger { 10 }); division_result.remainder == UnsignedBigInteger { 0 } && division_result.quotient != UnsignedBigInteger { 0 }; division_result = division_result.quotient.divided_by(UnsignedBigInteger { 10 })) {
```

Comment:
this...could use better formatting (and while you're at it, some constants):
```suggestion
        UnsignedBigInteger const ten { 10 };
        for (auto division_result = integer.divided_by(ten);
             division_result.remainder.is_zero() && !division_result.quotient.is_zero();
             division_result = division_result.quotient.divided_by(ten)) {
```

Reply:
Yeah. A monster.